src/graphics/program-lib/programs/standard.js

```diff
@@ -630,7 +630,10 @@ var standard = {
             }
         }
 
-        if (options.forceUv1) useUv[1] = true;
+        if (options.forceUv1) {
+            useUv[1] = true;
+            useUnmodifiedUv[1] = useUnmodifiedUv[1] ? useUnmodifiedUv[1] : true;
```

Comment:
isn't this line equivalent to just `useUnmodifiedUv[1] = true;` ?

Reply:
I don't think it is - the varying can be set to something else if it is used and there is a uv transformation - so it should skip the code that checks the useUnmodifiedUv and sets vUv1.

Reply:
oh wait - I fix this

Reply:
fixed.